pkg/i18n/english.go

```diff
@@ -842,6 +843,7 @@ func EnglishTranslationSet() TranslationSet {
 		LcStashStagedChanges:                "stash staged changes",
 		LcStashOptions:                      "Stash options",
 		NotARepository:                      "Error: must be run inside a git repository",
+		PromptToReturnFromSubprocess:        "Display confirmation when subprocess terminates",
```

Comment:
It doesn't look like this is used. If so, we should delete it

Reply:
I've found that typically when people want to add some string that needs to be i18'd they don't get this file first: they just assume that no matching string exists and add a new one themselves. So I'd consider this dead code and reckon we should remove it.